README.md

```diff
@@ -1 +1,80 @@
+
+If you need something immediate from me, this or [`/ad-hoc-irl`](#ad-hoc-irl) are the endpoints to use. For the most part I respond quickly but [there are](#context-switching) [instances](#meetings) where I may not. In those cases I will leave your message unread and circle back later. If I'm offline, however, all bets are off.
+
+### `/email`
+
+I treat my inbox with less urgency and tend to poll it when I'm in between tasks or looking for the next task to tackle (e.g. PR updates).
+
+### `/meetings`
+
+I like to know what a meeting is about before accepting it. If there's no description on a meeting request, don't be surprised if I ask for more details after receiving it.
+
+If I'm in a meeting I will try my absolute hardest to give you 100% of my attention. This is why I prefer not to bring my laptop to most meetings. My phone will also stay in my pocket unless an emergency call comes up. *Please call me out if you see me breaking either of these rules of mine.*
+
+Wrapping up the material before the allotted time is a _good thing_. It means we can all have some time back.
+
+### `/1-to-1s`
+
+This time is yours to tailor. The frequency and length of these meetings is up to you.
+
+Let's talk about anything that you feel would help you get better at what you do and anything that's getting in your way. If you can, spend some time before each meeting thinking about what you want to cover. Remember, this is about you -- don't squander it.
+
+### `/pair-programming`
+
+This is one of my favourites. Working through a problem with someone else is exciting. By the end of our time together we've created an artifact _and_ we most likely learned some things from each other.
+
+### `/ad-hoc-irl`
+
+If I'm at my desk feel free to grab my attention for anything you may want my input on. Don't be offended if I ask you to [wait a minute or two](#context-switching) if I'm in the middle of something.
+
+## Service level objectives
+
+### Transparency
+
+I like to get things out into the open. I believe people work at their best when they have full knowledge of what's going on around them.
+
+I'm open to any questions that you may have and will answer to the best of my abilities. I won't intentionally keep information from you.
+
+My calendar is as public as possible. Feel free to stalk it. I have found that a lot of context can be gained by seeing what people are up to. This ties back to having [detailed meeting descriptions](#meetings).
+
+I believe in Radical Candor and getting straight to the point. I will try my best to call out shortcomings that I see sans sugary exterior. Positive intent will always be assumed when approaching these conversations. _I encourage you to do the same with me._
+
+### Experience transfer
+
+I've spent a bunch of time in software development and have learnt things from others as well as from my own mistakes. I want to transfer that experience to others so they can learn the same things but in much less time than it took me.
```

Comment:
I'm not sure what this adds, it feels very general. How will you accomplish this?